app/workers/integration_worker.rb

```diff
@@ -0,0 +1,16 @@
+class IntegrationWorker
+  include Sidekiq::Worker
+
+  def perform(project_id, message)
+    project = Project.find(project_id)
+    project.integrations.each do |integration|
+      if integration.kind == 'mattermost'
+        Mattermost.send(integration.data['private_uri'],
```

Comment:
Later we can use STI for integrations, or use something like:

```ruby
integration.kind.contantize.send(...)
```

what you think?